### DISCUSSION TITLE
For the Stupid Password Rules at Iowa State | /en/2012/08/stupid-iastate-password-rules/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#777] [Comments]
https://yihui.org/en/2012/08/stupid-iastate-password-rules/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
Oh yes, thanks for the reminder. The seed was rpois(1, 1e8).

> Originally posted on 2012-08-07 13:39:30

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
I see. Thank you so much for being in the control group!

> Originally posted on 2012-08-07 13:41:21

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
Thanks for letting me know. I use Linux exclusively, but I did not try apg. I'm too frustrated to remember this 8-char password now :)

> Originally posted on 2012-08-07 13:47:20

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
I can imagine there is a field in their database which was created as CHAR(8).

> Originally posted on 2012-08-07 13:50:51

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
The problem is I have to guarantee Dropbox has been installed everywhere, or I'll have to login the Dropbox website to get that encrypted file, then I have to download the software to decrypt it. I'm totally fine on my own machine, but I do not have access to it all the time.  The only "portable" solution is to memorize the password, but for our system, the password expires again and again...

> Originally posted on 2012-08-07 15:51:12

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
Long live the statisticians!

> Originally posted on 2012-08-08 02:24:34

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
They have yet another mysterious rule which is your new password must not be similar to the old ones. I did shuffle my password for a couple of times, but failed for a few times due to the unknown "similarity measure".

> Originally posted on 2012-08-10 17:23:55

[a comment by an automated account is not shown]

**Comment by name-temp** [comment]
I know this post is well over 10 years old, but stumbled upon it while looking for something else, and wanted to shed some light on the weird requirements. The password requirements basically 100% match the IBM RACF requirements for their ALPHANUM symbol set (the default setting for *many* years), so more likely than not this password limitation was from ISU using a mainframe somewhere, that you had an account on, potentially indirectly.
https://www.newera.com/INFO/RACF_PW_Symbols.pdf

**Reply by yihui** [reply]
Good to know. Thank you for pointing out the possible source!
